types/googlemaps/reference/map.d.ts

```diff
@@ -603,15 +603,23 @@ declare namespace google.maps {
     /**
      * This object is returned from various mouse events on the map and overlays,
      * and contains all the fields shown below.
+     * https://developers.google.com/maps/documentation/javascript/reference/map#MapMouseEvent
      */
-    interface MouseEvent {
+    interface MapMouseEvent {
```

Comment:
Please update the version in the _index.d.ts_ file, apparently this is version [3.43.3](https://developers.google.com/maps/documentation/javascript/releases).

>   *   Renamed the `google.maps.MouseEvent` interface to `google.maps.MapMouseEvent` and added a `domEvent` property, providing direct access to the underlying event from the DOM.

Reply:
Updated as requested.

Reply:
The documentation says that this version should not have a patch version, as the build process adds that.  This is probably why the CI failed?  If I get time I'll create an account and login to see the specific reason(s).